Tanmayee Gujar Homework 3 CP1 2021

TanmayeeGujar opened the pull request:
Added the circuit link for HW1 adder, and also added the python notebooks for the phase calculator and full adder

TanmayeeGujar committed:
Added files for full adder and phase calculator

TanmayeeGujar committed:
Updated the ReadMe.md file

TanmayeeGujar committed:
Added link to circuit for HW#1

TanmayeeGujar committed:
Merge pull request #1 from TanmayeeGujar/TanmayeeGujar-HW3CP12021

Added files for full adder and phase calculator

admarga commented:
Great work! There are some spelling errors in your README though.

admarga commented on README.md:
```diff
@@ -1,44 +1,32 @@
-Put in a discussion here.
-
-e.g. My code could compute 1+2=3, however it took the QC 3 seconds and out of 1024 trials it did it wrong in 2% of the cases. So it might be not the best choice for this task. However, they are awesome for optimizing the [traveling salesman problem](https://xkcd.com/285/).
+My adder got the accurate sum with a prabability of 0.492, and it got pretty close (1.0188202912233635) to the value of π/3 (= 1.047197551196597) when run on a quantum computer. It is not fully accurate because this is not an ideal problem to be solved on a quantum conputer.
```
*computer.

TanmayeeGujar replied:
Will create an issue and fix it. Thank you!

TanmayeeGujar replied:
Fixed!

admarga reviewed (changes requested):
Read over the README and adjust the tenses to the pronouns.

TanmayeeGujar committed:
Fixed typo and resolved issue #16

admarga reviewed (changes requested):
Add References!!!!

TanmayeeGujar committed:
Added References

lmeiler reviewed (commented):
I think it looks great. One improvement that I would suggest would be to go into more detail in your Markdown documentation. I think it might be useful to write a small paragraph detailing the purpose and results of the notebook at the beginning and end of it.

TanmayeeGujar committed:
Adder.ipynb file now runs on local Docker image

The new updated Adder_Docker.ipynb notebook now is able to run the 1+1+1=3 adder locally on Docker image

TanmayeeGujar committed:
Added pdf document for HW4

TanmayeeGujar commented:
> I think it looks great. One improvement that I would suggest would be to go into more detail in your Markdown documentation. I think it might be useful to write a small paragraph detailing the purpose and results of the notebook at the beginning and end of it.

I will work on it, thank you!

laserlab commented on Adder_Docker.ipynb:
```diff
@@ -0,0 +1,261 @@
+   "outputs": [],
+   "source": [
+    "#IBMQ.delete_account()\n",
+    "IBMQ.save_account('7d8a2b3e50702a93549aef58c63889439f2097b22aa826ddd3288b29ec6e0c39323697aec7a13cc6ebcad9b54ea264568443bd100b6b22807075618946ee8724', overwrite=True)\n",
```
do not post api keys!

laserlab commented on README.md:
```diff
@@ -1,44 +1,39 @@
-e.g.
 
-If I would do X and Y my implementation could even count to -3.
+## References 
```
Please use inline references so it is clear which statement / code is from which source!

laserlab commented:
> Read over the README and adjust the tenses to the pronouns.

Good solution to open an issue for a review request

laserlab reviewed (changes requested):
- references have to be used at the location of the statement / code
